tests/integration/venv_ITs/test_issue_2065.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import json
+import os
+import subprocess
+from textwrap import dedent
+
+from pex.common import safe_open
+from pex.testing import make_env, run_pex_command
+from pex.typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any
+
+
+def test_venv_pex_script_non_hermetic(tmpdir):
+    # type: (Any) -> None
+
+    # A console script that injects an element in the PYTHONPATH.
+    ot_simulator_src = os.path.join(str(tmpdir), "src")
```

Comment:
As named, this simulates OpenTelemetry auto-instrumentation - actually turning the manual test  detailed here https://github.com/pantsbuild/pex/issues/2065#issuecomment-1444451584 into an IT seemed too fragile / likely flaky.